### DISCUSSION TITLE
Rename getHeaders() to getHeadings()

### DISCUSSION THREAD

**Discussion by felixsanz** [withastro/roadmap#208] [Proposal]
Docs say:
```
getHeaders(): An async function that returns the headers of the Markdown file.
The response follows this type: { depth: number; slug: string; text: string }[].
```

But according to the HTML spec, these are **headings**. The only part of the spec where the word "headers" appear is in the context of `<header>` tag. MDN also uses `headings` to refer the `<h1-6>` tags.

I know this will be a breaking change and everyone understand it as it is, but i would rename it anyway while Astro is still in beta and these changes do less damage now.

What do you think?

**Comment by ElianCodes** [comment]
I do agree, It's also a good Idea to do this while in beta. Just wondering if keeping 'both' methods available would be a better option for now

**Reply by felixsanz** [reply]
yeah, like with <Markdown> component, a deprecation notice helps!

**Comment by FredKSchott** [comment]
+1 from myself and @natemoo-re! We followed up on next steps here in Discord: https://discord.com/channels/830184174198718474/872579324446928896/987039940187668521

**Comment by felixsanz** [comment]
Completed in https://github.com/withastro/astro/pull/3627 and https://github.com/withastro/docs/pull/791
